paasta_tools/cli/cmds/spark_run.py

```diff
@@ -110,7 +111,12 @@ def add_subparser(subparsers):
         "--image",
         help="Use the provided image to start the Spark driver and executors.",
     )
-
+    list_parser.add_argument(
+        "-a",
+        "--autogen",
```

Comment:
Looks like autogen is an integer? If so, add the param type, `type=int`.
`autogen` seems to be used as a weight, which has range from (1, 100). [Ref.](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).

Use `choice(1, 100)` too, in that case. Ref.: https://docs.python.org/3/library/argparse.html#choices